src/content/en/2020/http2.md

```diff
@@ -126,52 +185,54 @@ Figure 22.6 shows the fraction of HTTP/2 or gQUIC requests per page, ordered by
+  description="A line chart comparing the fraction of first-party HTTP/2 requests with third-party HTTP/2 or gQUIC requests. The chart orders he websites by fraction of HTTP/2 requests. 45% of websites have no HTTP/2 first-party requests. Over half of websites serve third-party requests only over HTTP/2 or gQUIC. 80% of websites have 76% or more third-party HTTP/2 or gQUIC requests.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSOkWXtrbMfTLdhlKbBGDjRU3zKbnCQi3iPhfuKaFs5mj4smEzInDCYEnk63gBdgsJ3GFk2gf4FOKCU/pubchart?oid=1409316276&format=interactive",
   sql_file="http2_%_1st_party_vs_3rd_party.sql",
   sheets_gid="733872185"
 )
 }}
 
-Figure 22.8 shows the distribution of known third-party HTTP/2 or gQUIC requests by content-type. For example 90% of websites serve 100% of third party fonts and audio over HTTP/2 or gQUIC, only 5% over HTTP/1 and 5% are a mix. The majority of third-party assets are either scripts or images, and are solely served over HTTP/2 or gQUIC on 60% and 70% of web sites respectively.
+Figure 22.8 shows the distribution of known third-party HTTP/2 or gQUIC requests by content-type. For example 90% of websites serve 100% of third party fonts and audio over HTTP/2 or gQUIC, only 5% over HTTP/1 and 5% are a mix. The majority of third-party assets are either scripts or images, and are solely served over HTTP/2 or gQUIC on 60% and 70% of websites respectively.
 
 {{ figure_markup(
   image="http2-third-party-http2-usage-by-content-type.png",
-  caption="The fraction of known third-party HTTP/2 or gQUIC requests by content-type per web site.",
+  caption="The fraction of known third-party HTTP/2 or gQUIC requests by content-type per website.",
   description="A bar chart comparing the fraction of third-party HTTP/2 requests by content-type. All third-party requests are served over HTTP/2 or gQUIC for 90% of audio and fonts, 80% of css and video, 70% of html, image and text and 60% of scripts.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSOkWXtrbMfTLdhlKbBGDjRU3zKbnCQi3iPhfuKaFs5mj4smEzInDCYEnk63gBdgsJ3GFk2gf4FOKCU/pubchart?oid=1264128523&format=interactive",
   sheets_gid="419557288",
   sql_file="http2_%_1st_party_vs_3rd_party_by_type.sql"
 )
 }}
 
-Figure 22.9 looks at protocol usage by the third party category. Ads, analytics, Content Delivery Network resources (CDN), and tag-managers are predominantly served over HTTP/2 or gQUIC. Customer-success and marketing content is more likely to be served over HTTP/1.
+{# TODO(authors, analysts): Is this breakdown specifically looking at mobile? I don't see a notice that all data is mobile unless otherwise noted, so it's worth clarifying here. #}
```

Comment:
@dotjs @bazzadp

Reply:
Look to be mobile. We should add a note in the intro to say, "where results are similar between desktop and mobile, stats are presented from the mobile dataset." or something like that.

Reply:
Yes As @bazzadp told me it's a mobile first web. I'll add a note in the intro.